### ISSUE TITLE
PostGis + multi_polygon

### ISSUE THREAD

**Issue by jak4** [rgeo/rgeo#77]
Hi,

I'm having a rather weird issue regarding PostGIS (9.1), rgeo (0.3.20) and rego-activerecord (1.0.0).

I have a column named shape (MultiPolygon,4326) with MultiPolygons in my PostGIS-DB. 
My model has this factory definition: set_rgeo_factory_for_column(:shape, RGeo::Geographic.spherical_factory(:srid => 4326)).

To display those multi-polygons I extract the lat/lngs from the points and return them to a leaflet.js map. This is working perfectly in development. However in production the factory seems to be ignored, since the returned points for the polygon/multi-polygon are of type: #RGeo::Cartesian::PointImpl:0x000000068f9b60.

But its even better. If I perform the same operations in the rails console in production mode on the same system everything works as expected, thus the points are of type SphericalImpl.

Does anybody have a clue as to why this is happening?

TLDR: Spherical factory setting for column shape (multi-polygon) is ignored in production but not in development or in production rails console.

Hannes
